Contentstack.Core/Internals/ContentstackRegion.cs

```diff
@@ -13,14 +13,25 @@ public enum ContentstackRegion
         /// <summary>
         /// To specify EU region.
         /// </summary>
-        EU
+        EU,
+        /// <summary>
+        /// To specify EU region.
+        /// </summary>
+        Azure_EU,
+        /// <summary>
+        /// To specify EU region.
+        /// </summary>
+        Azure_NA
+
     }
 
 
     internal enum ContentstackRegionCode
     {
         us,
 
-        eu
+        eu,
+
+        azure
```

Comment:
@AnkurContentstack Please check the endpoint for Azure and add code accordingly

Reply:
here we can not added AZURE-NA because **-** is not allowed in enum